src/index.ts

```diff
@@ -34,15 +29,14 @@ import { fromBase64, toBase64 } from 'b64-lite'
 import * as pack from '../package.json'
 import { ReadTransaction } from './ReadTransaction'
 import { WriteTransaction } from './WriteTransaction'
-import { JSONQuery } from './query'
+import { JSONQuery, Entity, EntityList } from './models'
```

Comment:
One thing I noticed from playing around yesterday is the need to expose more things other than `Client`, like `JSONQuery`. Probably other stuff too.

Reply:
Ah yeh good call! I'll those in before merging.